srcpkgs/gtk4/template

```diff
@@ -6,9 +6,9 @@ build_style=meson
 build_helper="gir"
 configure_args="-Dman-pages=true -Dgtk_doc=true -Dbuild-tests=false
  -Dbuild-testsuite=false -Dbuild-examples=false
- $(vopt_feature broadway broadway-backend)
- $(vopt_feature x11 x11-backend)
- $(vopt_feature wayland wayland-backend)
+ $(vopt_bool broadway-backend)
```

Comment:
doesn't this need `broadway broadway-backend`? that's what the build option is

Reply:
Good catch, I messed that one up.